template/src/css/main.css

```diff
@@ -33,14 +33,24 @@
 }
 
 .border-danger {
-  border: 1px solid var(--red);
+  border: 1px solid var(--red) !important;
```

Comment:
why is this needed now? I'd preferer if css doesn't contain any `!important` as it's a sign of incorrect cascade.

Reply:
Fixed